### PR TITLE
ISPN-14603 SUNION SUNIONSTORE commands

### PR DESCRIPTION
**Pull request by rigazilla**
https://issues.redhat.com/browse/ISPN-14603

based on ~~#11048~~

### CODE REVIEW

File: multimap/src/main/java/org/infinispan/multimap/impl/SetBucket.java [hunk 1]
```diff
@@ -52,8 +52,8 @@ Collection<MultimapObjectWrapper<V>> getWrappedValues() {
       return new ArrayList<>(values);
    }
 
-   public Set<V> values() {
-      return values.stream().map(MultimapObjectWrapper::get).collect(Collectors.toSet());
+   public Set<MultimapObjectWrapper<V>> values() {
+      return values;
```
Comment (jabolina) [inline multimap/src/main/java/org/infinispan/multimap/impl/SetBucket.java:56]: I think we need to copy here.

**Review by jabolina — APPROVED** [review]
LGTM, just one small thing.

**Comment by jabolina** [comment]
There are couple issues in CI.

File: multimap/src/main/java/org/infinispan/multimap/impl/SetBucket.java (hunk 1 above)
Comment (rigazilla) [reply multimap/src/main/java/org/infinispan/multimap/impl/SetBucket.java]: I removed that method, it's not needed actually. 
I replaced it with a toList method, which returns a copy in a list

**Review by karesti — APPROVED** [review]

**Review by karesti — CHANGES_REQUESTED** [review]
the list of commands has not been updated (docs)

**Review by karesti — APPROVED** [review]
I'm happy @rigazilla

**Comment by rigazilla** [comment]
share your happiness with infinispan/main @karesti !

**Pull request merged**